Economy.md

```diff
@@ -0,0 +1,89 @@
+# EssentialsX Economy
+EssentialsX comes with a simple, easy-to-use economy system. The built in economy API will redirect to any other detected economy system such as vault. This means you can use EssentialsX Economy as a economy API instead of Vault if your plugin is dependent on EssentialsX.
+
+## Commands
+There are a number of commands for use in-game, including but not limited to:
+   - /eco, /economy: Allows you to take, give, or reset a player's money.
+   - /sell: Sells an item and amount at the price specified in the worth.yml.
+   - /balance, /bal, /money: Displays the current balance of a player.
+   - /baltop, /balancetop: Displays a list of the richest people.
+   - /pay: Pays a specified player from your balance.
```

Comment:
It might be better to wrap the commands in backticks (`) for consistency with other EssentialsX documentation.

For example, `/baltop`, `/balancetop` can be written in Markdown as:

```markdown
`/baltop`, `/balancetop`
```